test/fixtures/full_config.yml

```diff
@@ -534,7 +543,7 @@ Layout/IndentationConsistency:
   EnforcedStyle: normal
   SupportedStyles:
   - normal
-  - rails
+  - indented_internal_methods
   Reference:
   - https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions
```

Comment:
Is this still relevant? Shouldn't we be pointing to our own style guide instead of the Rails one?

Reply:
This is unrelated with this PR, but for the reference https://github.com/rubocop/rubocop/pull/7059